eve/main.yml

```diff
@@ -30,11 +33,7 @@ stages:
     steps:
     - MasterShellCommand:
         name: Replace upstream image with `-ci` variant
-        command: "sed '/^FROM/ s/$/-ci/' images/warp10/Dockerfile"
-        workdir: "%(prop:master_builddir)s/build/"
```

Comment:
don't know why but the `workdir` parameter is not available on the `MasterShellCommand` step, therefore I've added the property directly on the command itself